packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -321,6 +321,9 @@ class KeepAliveNotification extends Notification {
 class KeepAliveHandle extends ChangeNotifier {
   /// Trigger the listeners to indicate that the widget
   /// no longer needs to be kept alive.
+  ///
+  /// This method does not call [dispose]. When the handle is not needed
+  /// anymore, it must be [dispose]d regardless of whether notifying listeners.
```

Comment:
Should we just fix this by deprecating release and overriding `dispose` to call `notifyListeners` before `super.dispose`?

Reply:
I think that makes sense. I wasn't clear on what if anything that would break.

Reply:
Do you want me to do that in this PR or should we file an issue/follow up?

Reply:
I would prefer doing it in this PR, but I would be fine either way.